test/service/test_response.py

```diff
@@ -1290,6 +1290,18 @@ def test_file_response(self):
         file = one(one(keyword_response['hits'])['files'])
         self.assertElasticsearchResultsEqual(file, expected_file)
 
+    def test_faulty_filter(self):
```

Comment:
```suggestion
    def test_nonexistant_filter_facet(self):
```

Reply:
Seems to cover what `test_bad_single_filter_facet_of_sample` already covers. If so, please delete, otherwise please explain what the coverage difference is.

Reply:
You're right. I looked for and failed to find this test before. My mistake.